index.bs

```diff
@@ -1751,7 +1767,7 @@ an |ownership type|, a |serialization internal map| and a |realm|:
 1. Let |known object| be <code>true</code>, if |value| is in the
    |serialization internal map|, otherwise <code>false</code>.
 
-1. Let |child ownership| be "<code>none</code>".
+1. Let |ownership type| be "<code>none</code>".
```

Comment:
I don't think this change is wanted and it wasn't here last time. Note that it will override the `ownership type` argument of the `serialize as a remote value` method.

Reply:
I found it confusing that we were calling the argument `|child ownership|` in the algorithms, when it's not related to any children. So I'd like to keep that part at least. Replacing the passed in value is technically correct: we use the original `ownership type` variable once in `handle for an object` and after that it's an error to ever use the original value again for future calls (in Rust parlance, we move that value into the function). But maybe this is also confusing.

Reply:
Could you maybe be fine with splitting off this particular change to a different PR? It's not really related to adding support for `NodeList` and `HTMLCollection`.